src/containers/Main/Main.js

```diff
@@ -138,7 +138,8 @@ class Main extends Component {
   render() {
 
     let not_logged_in = null;
-    if (!this.props.token) {
+    // consider accessing localstorage only once during componentDidMount
```

Comment:
I was wondering about that!

Reply:
Yep! `props.token` is technically unused for now, but I'm leaving it in as a reminder that I want to change it to be saved to redux and access that instead of directly pulling from localstorage in the middle of the code like it is in this pull request.